functions-and-scope.js

```diff
@@ -13,10 +13,28 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorg ik ervoor dat dit ook werkt wanneer de array 100 entries bevat?
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
-
+/*
+* Stap1 maak een functie heet "studentsHasGraduated"
+* Stap2 declareren een variabel om afgestudeerde studenten te kunnen rekenen "let a=0;"
+* Stap3 schrijft een for loop om alle grades te kunnen lopen "for (let i = 0; i < grades.length; i++)"
+* Stap4 geeft een if condition om afgestudeerde studenten te kiezen "if (grades[i] > 7)"
+* Stap5 declareren een variabel om als if condition true is dan het te kunnen 1 getal plussen "const b = 1;"
+* Stap6 variabel a + variabel b dan geeft to a om correct afgestudeerde studenten te kunnen krijgen "a = a + b;"
+* Stap7 return variabel a om kunnen function te aanroepen "return a;"
+* */
 // ---- Verwachte uitkomst: 6
-
-
+function studentsHasGraduated(){
+    let a=0;
+    for (let i = 0; i < grades.length; i++) {
+        if (grades[i] > 7){
+            const b = 1;
+            a = a + b;
```

Comment:
Je kan dit versimpelen naar
```suggestion
            a++;
```